tests/installation/first_boot.pm

```diff
@@ -10,6 +10,11 @@ sub run() {
         return;
     }
 
+    if (check_var('DESKTOP', 'awesome')) {
```

Comment:
well, that check_var('DESKTOP', 'awesome') should be intentional to go away of the test since it's NOAUTOLOGIN test :)